internal/store/installation.go

```diff
@@ -127,19 +174,25 @@ func (sqlStore *SQLStore) CreateInstallation(installation *model.Installation) e
 
 // UpdateInstallation updates the given installation in the database.
 func (sqlStore *SQLStore) UpdateInstallation(installation *model.Installation) error {
-	_, err := sqlStore.execBuilder(sqlStore.db, sq.
+	envJSON, err := json.Marshal(installation.MattermostEnv)
+	if err != nil {
+		errors.Wrap(err, "unable to marshal mattermost env")
```

Comment:
nit: "mattermost env" -> "MattermostEnv" just so that it's clear when this error is printed that the issue is with the field MattermostEnv and not some generic "mattermost environment"